GitHubPages.gitignore

```diff
@@ -0,0 +1,17 @@
+# This .gitignore is appropriate for repositories deployed to GitHub Pages and using
+# a Gemfile as specified at https://github.com/github/pages-gem#conventional
+
+# Basic Jekyll gitignores (synchronize to Jekyll.gitignore)
+_site/
+.sass-cache/
+.jekyll-cache/
+.jekyll-metadata
+
+# Additional Ruby/bundler ignore for when you run: bundle install
+/vendor
+
+# Specific ignore for GitHub Pages
+# GitHub Pages will always use its own deployed version of pages-gem 
+# This means GitHub Pages will NOT use your Gemfile.lock and therefore it is not
+# counterproductive to check this file into the repository.
```

Comment:
Possible typo:
> "therefore it is **not** counterproductive"
-> "therefore it is counterproductive"

Reply:
Thank you. 978e3010329eff589d2b9b140cb8af30f4480065

Reply:
P.S. "I have written at least 240,000 words..." this is only a few Tweets a day

Reply:
Haha, thanks! That's currently a very rough draft (did a word count on some random files). I am going to go through all my hard-drives & get a solid concrete number soon. Tweets / day is a great measurement, 240k words comes out to only 65 words per day over a decade. Definitely much higher. Really appreciate the feedback, thank you.